src/pnpq/apt/protocol.py

```diff
@@ -134,6 +134,22 @@ def from_bool(cls, toggle: bool) -> "EnableState":
         return cls.CHANNEL_DISABLED
 
 
+@enum.unique
+class StopMode(int, Enum):
+    """Used in MSMSG_MOT_MOVE_STOP."""
+
+    IMMEDIATE_STOP = 0x01
+    CONTROLLED_STOP = 0x02
```

Comment:
Removing the redundant "STOP." It's probably best not to merge this suggestion but update it on your machine so that you can update references to it at the same time.

```suggestion
    IMMEDIATE = 0x01
    CONTROLLED = 0x02
```